tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_eltwise_polygamma.py

```diff
@@ -42,7 +42,7 @@ def run_eltwise_polygamma_tests(
 
     assert len(tt_result.shape) == len(ref_value.shape)
     assert tt_result.shape == ref_value.shape
-    assert_with_pcc(ref_value, tt_result, 0.99)
+    assert_with_pcc(ref_value, tt_result, 0.98)
```

Comment:
Is this due to making the scalar tile bfp8?

I have also added mixed precision support to bcast, so these test cases should also work if the scalar tile is bfloat16 now (once #8627 is merged)

Reply:
Yes @tt-aho